### DISCUSSION TITLE
How to use the program programmatically?

### DISCUSSION THREAD

**Discussion by BazzaRB** [Anjok07/ultimatevocalremovergui#1071] [General]
Hi, i was wondering how i could use a python program to set the params and send an audiofile to the program, and recive the filepath to the audio file without the music on it manualy in python if that makes sense. Since i need to do this for video files, and it would make it easier to make an automated program for this. So does this program have a library i can use or simulare?

**Comment by 3megabytesofhotram** [comment]
You can use this package https://pypi.org/project/audio-separator/
